mac

```diff
@@ -190,6 +190,14 @@ if [ "$brew_last_change" -le "1471233600" ]; then
   print_done
 fi
 
+# Fix permission issue on /usr/local on OS 10.12
+if [ ! -w /usr/local ]; then
```

Comment:
@devpuppy was the bug that `/usr/local` wasn't writeable; or that a particular `n` version wasn't (i.e. `/usr/local/n/versions/`)?

This would not catch the case where `/usr/local` is writeable; but `/usr/local/n/*` isn't. IDK from the bug report if that was the case or not.